pkg/linter/linterTypes.go

```diff
@@ -67,15 +69,23 @@ func (lint *basicLinter) enableByDefault() bool {
 	return lint.enable
 }
 
-func (lint *basicLinter) string(lintDesc string) string {
-	findingsRes := make([]string, len(lint.findings))
+func (lint *basicLinter) string(lintDesc string, printAll bool) string {
+	findingsResAll := make([]string, len(lint.findings))
 	for i, thisFinding := range lint.findings {
-		findingsRes[i] = thisFinding.string()
+		findingsResAll[i] = thisFinding.string()
+	}
+	sort.Strings(findingsResAll)
+	var suffix string
+	var findingRes []string
+	if !printAll && len(lint.findings) > numFindingToPrint {
+		findingRes = findingsResAll[:numFindingToPrint]
+		suffix = fmt.Sprintf("\n...and %d more\n", len(lint.findings)-numFindingToPrint)
+	} else {
+		findingRes = findingsResAll
 	}
-	sort.Strings(findingsRes)
 	header := fmt.Sprintf("%q %s\n", lintDesc, issues) +
-		strings.Repeat("~", len(lintDesc)+len(issues)+3) + "\n"
-	return header + strings.Join(findingsRes, "\n")
+		strings.Repeat("~", len(lintDesc)+len(issues)+numFindingToPrint) + "\n"
```

Comment:
```golang
	header := fmt.Sprintf("%q %s\n", lintDesc, issues)
	header += strings.Repeat("~", len(header)-1) + "\n"
```

Reply:
or maybe the `issues` const is no longer needed

Reply:
right. done